airflow/providers/openlineage/plugins/listener.py

```diff
@@ -51,6 +57,16 @@ def __init__(self):
         self.log = logging.getLogger(__name__)
         self.extractor_manager = ExtractorManager()
         self.adapter = OpenLineageAdapter()
+        self._selective_enable = conf.getboolean("openlineage", "selective_enable", fallback=False)
+
```

Comment:
After you merge the current main that includes #38033, there will be no need to assign this as an attibute, as it can be added to conf.py and cached.

Reply:
Moved logic to `conf.py` and related.